newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -483,6 +489,25 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
       </Line>
     );
 
+  const getExtremeAllowedScoreText = (currentLeaderboard: Leaderboard) => {
+    const maxOrMin = currentLeaderboard.sort === 'ASC' ? 'min' : 'max';
+    if (
+      currentLeaderboard.extremeAllowedScore === null ||
+      currentLeaderboard.extremeAllowedScore === undefined
+    ) {
+      return '';
+    }
+    let customizedScoreText = currentLeaderboard.extremeAllowedScore;
+    if (currentLeaderboard.customizationSettings) {
+      customizedScoreText = formatScore(
+        // $FlowIgnore
```

Comment:
Can you explain in the comment why this is safe to ignore?

Reply:
Actually, we don't need to ignore 😅